patches/chrome-android-java-src-org-chromium-chrome-browser-contextmenu-ChromeContextMenuItem.java.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
+index 7daa27cc9cfa48fb1cb4314d06c1f2de299fc38b..bf84c9e92c17da81530397dad475fd2a971ebb60 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
+@@ -42,7 +42,7 @@ class ChromeContextMenuItem {
+             Item.SHOP_IMAGE_WITH_GOOGLE_LENS, Item.SHARE_IMAGE, Item.DIRECT_SHARE_IMAGE, Item.CALL,
+             Item.SEND_MESSAGE, Item.ADD_TO_CONTACTS, Item.COPY, Item.SAVE_VIDEO,
+             Item.OPEN_IN_CHROME, Item.OPEN_IN_NEW_TAB_IN_GROUP, Item.SHARE_HIGHLIGHT,
+-            Item.REMOVE_HIGHLIGHT, Item.LEARN_MORE})
++            Item.REMOVE_HIGHLIGHT, Item.LEARN_MORE, Item.COPY_CLEAN_LINK})
+     @Retention(RetentionPolicy.SOURCE)
+     public @interface Item {
+         // Values are numerated from 0 and can't have gaps.
+@@ -88,9 +88,9 @@ class ChromeContextMenuItem {
+         // Shared Highlighting options
+         int SHARE_HIGHLIGHT = 32;
+         int REMOVE_HIGHLIGHT = 33;
+-        int LEARN_MORE = 34;
++        int LEARN_MORE = 34;int COPY_CLEAN_LINK = 35;
```

Comment:
Sorry, I meant `COPY_CLEAN_LINK` and `NUM_ENTRIES` in one line, so it would be 1 change instead of 2.